flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -35,7 +36,8 @@
 /**
  * Context object with optional arguments for a Flink Scan.
  */
-class ScanContext implements Serializable {
+@Internal
+public class ScanContext implements Serializable {
```

Comment:
This is made public because it is accessed by `ContinuousSplitPlannerImpl` class in this PR.